packages/cli/openapi-parser/src/openapi/v3/extensions/getExamplesFromExtension.ts

```diff
@@ -3,11 +3,13 @@ import { RawSchemas } from "@fern-api/yaml-schema";
 import { OpenAPIV3 } from "openapi-types";
 import { getExtensionAndValidate } from "../../../getExtension";
 import { AbstractOpenAPIV3ParserContext } from "../AbstractOpenAPIV3ParserContext";
+import { OperationContext } from "../converters/contexts";
 import { RedoclyCodeSampleArraySchema, RedoclyCodeSampleSchema } from "../schemas/RedoclyCodeSampleSchema";
 import { OpenAPIExtension } from "./extensions";
 import { FernOpenAPIExtension } from "./fernExtensions";
 
 export function getExamplesFromExtension(
+    operationContext: OperationContext,
```

Comment:
@abvthecity as you add more code to the importer, one thing i've wanted to do aspirationally is build a `logger` that just keeps the context of where it is (paths -> "/users" -> post) so that whenever you log it gives you a location.